src/components/Modals/AddContactModal.jsx

```diff
@@ -64,7 +71,10 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
     const userObject = {
       givenName: event.target.addUserGivenName.value,
       familyName: event.target.addUserFamilyName.value,
-      webId: event.target.addWebId.value
+      webId: event.target.addWebId.value,
+      pod: event.target.pod.value,
```

Comment:
`pod` is redundant here. It's already part of the object utilized in contacts.

Reply:
I will work on the unit tests. So there will be another way to add 'relation' and 'relationshipStatus' to the contact properties?

Reply:
Was thinking about utilizing the contact list for those, especially with relationship status. However, we could potentially include the option to set relationships in the sign up flow on the sign up page.